packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -120,145 +120,75 @@ const ClinicalResultBloodPressure = (props) => {
-      { truncated: isTruncated },
-      { 'status-in-error': compareStatusIsInError.systolic || compareStatusIsInError.diastolic },
-    ]);
+  const systolicResult = sanitizeResult(systolic);
+  const diastolicResult = sanitizeResult(diastolic);
+
+  const conceptDisplayElement = createConceptDisplays({
+    originalSystolic: systolicResult.conceptDisplay,
+    originalDiastolic: diastolicResult.conceptDisplay,
+    systolic: systolicResult.cleanedConceptDisplay,
+    diastolic: diastolicResult.cleanedConceptDisplay,
+  });
+
+  const datetimeDisplayElement = createDatetimeDisplays({
+    originalSystolic: systolicResult.datetimeDisplay,
+    originalDiastolic: diastolicResult.datetimeDisplay,
+    systolic: systolicResult.cleanedDatetimeDisplay,
+    diastolic: diastolicResult.cleanedDatetimeDisplay,
+  });
+
+  const hasModifiedIcon = (systolicResult.isModified) || (diastolicResult.isModified);
+  const hasCommentIcon = (systolicResult.hasComment) || (diastolicResult.hasComment);
+  const hasUnverifiedIcon = (systolicResult.isUnverified) || (diastolicResult.isUnverified);
+
+  const decoratedResultDisplay = (
+    <>
+      <BloodPressureDisplay result={systolicResult} hideUnit={hideUnit} id={id} type="Systolic" diastolicUnit={diastolicResult.cleanedUnit} />
+      <span key={`Observation-Separator-${(systolic) ? systolic.eventId : diastolic.eventId}`} className={cx('result-display-separator')}>/</span>
+      <BloodPressureDisplay result={diastolicResult} hideUnit={hideUnit} id={id} type="Diastolic" />
+    </>
+  );
 
-    clinicalResultBloodPressureDisplay = (
+  const modifiedIconElement = hasModifiedIcon && !hasUnverifiedIcon ? (<IconModified className={cx('icon-modified')} />) : null;
+  const commentIconElement = hasCommentIcon && !hasUnverifiedIcon ? (<IconComment className={cx('icon-comment')} />) : null;
+  const unverifiedIconElement = hasUnverifiedIcon ? (<IconUnverified className={cx('icon-unverified')} />) : null;
+  let iconGroupDisplayElement = null;
+  if (hasModifiedIcon || hasCommentIcon || hasUnverifiedIcon) {
+    iconGroupDisplayElement = (
       <React.Fragment>
-        <div className={decoratedResultClassnames}>
-          <div className={cx('result-display')}>
-            {decoratedResultDisplay}
-            {isTruncated ? null : !hideAccessoryDisplays && iconGroupDisplayElement}
-          </div>
-          {isTruncated ? !hideAccessoryDisplays && iconGroupDisplayElement : null}
-        </div>
-        {!hideAccessoryDisplays && conceptDisplayElement}
-        {!hideAccessoryDisplays && datetimeDisplayElement}
+        {modifiedIconElement}
+        {commentIconElement}
+        {unverifiedIconElement}
       </React.Fragment>
     );
   }
 
+  const decoratedResultClassnames = cx([
+    'decorated-result-display',
+    { truncated: isTruncated },
+    { 'status-in-error': systolicResult.statusInError || diastolicResult.statusInError },
+  ]);
+
+  clinicalResultBloodPressureDisplay = (
```

Comment:
can we move variable instantiation here? otherwise it's all the way at the top
```suggestion
  const clinicalResultBloodPressureDisplay = (
```

Reply:
Fixed here https://github.com/cerner/terra-clinical/pull/648/commits/c64377629f953ac331135eeef3730d8dc9ccb321